rclcpp/src/rclcpp/executors/static_executor_entities_collector.cpp

```diff
@@ -87,9 +88,16 @@ StaticExecutorEntitiesCollector::fini()
   exec_list_.clear();
 }
 
+std::shared_ptr<void>
+StaticExecutorEntitiesCollector::take_data()
+{
+  return nullptr;
+}
+
```

Comment:
The `StaticExecutorEntitiesCollector` would have some synchronization issues if we would try to use it in a `MultiThreadedExecutor`, but we're not so this is ok for the moment.